tests/Jwt/FirebaseParserTest.php

```diff
@@ -51,32 +50,28 @@ protected function setUp()
         ];
     }
 
+    /**
+     * @expectedException \Equip\Auth\Exception\InvalidException
+     * @expectedExceptionCode \Equip\Auth\Exception\InvalidException::CODE_TOKEN_EXPIRED
```

Comment:
Anything stopping us from upgrading to PHPUnit >= 5.2 and transitioning to [this practice](https://thephp.cc/news/2016/02/questioning-phpunit-best-practices) as I [suggested earlier](https://github.com/equip/auth/pull/3#issuecomment-181497866)?